src/pretix/control/templates/pretixcontrol/checkin/bulk_revert_confirm.html

```diff
@@ -0,0 +1,25 @@
+{% extends "pretixcontrol/items/base.html" %}
+{% load i18n %}
+{% load bootstrap3 %}
+{% block title %}{% trans "Delete check-ins" %}{% endblock %}
+{% block inside %}
+	<h1>{% trans "Delete check-ins" %}</h1>
+	<form action="{% url "control:event.orders.checkinlists.bulk_action" event=request.event.slug organizer=request.event.organizer.slug list=checkinlist.pk %}" method="post" class="form-horizontal" data-asynctask>
+		{% csrf_token %}
+        {% for k, l in request.POST.lists %}
+            {% for v in l %}
+                <input type="hidden" name="{{ k }}" value="{{ v }}">
+            {% endfor %}
+        {% endfor %}
+		<p>{% blocktrans with count=cnt %}Are you sure you want to permanently delete the check-ins of <strong>{{ count }} tickets</strong>?{% endblocktrans %}</p>
```

Comment:
Not sure if it is a rare edge case, but what if count==1? Probably not worth the effort. And using „ticket(s)“ instead of „tickets“ does probably do more harm than help.

Reply:
TIL `{% plural %}`! Probably had the wrong search terms around ngettext and django templates. Thanks!